apps/admin/components/Tables/utils.js

```diff
@@ -38,22 +37,22 @@ export const tableStyles = {
   selectableRow: css({
     '& td': {
       transition: 'border-color 0.2s',
-      borderTop: `1px solid ${colors.secondary}00`,
-      borderBottom: `1px solid ${colors.secondary}00`,
+      borderTop: `1px solid ${'var(--color-secondary)'}`,
+      borderBottom: `1px solid ${'var(--color-secondary)'}`,
```

Comment:
@trm217 FYI, this was a weird one, where `00` would be appended to a color string (which obviously doesn't work with a var). No easy way to replicate this behavior but since it's in the admin app I think it's fine to ignore?